src/main/scala/uk/gov/hmrc/play/it/MicroServiceEmbeddedServer.scala

```diff
@@ -70,65 +66,6 @@ trait ExternalServiceOrchestrator extends StartAndStopServer {
   }
 }
 
-trait EmbeddedServiceOrchestrator extends ResourceProvider with StartAndStopServer {
```

Comment:
@frankoid added the following comment on the commit:

They use classes removed between Play 2.3 and 2.5.

ScalaTest+Play has mixins that can start a Play HTTP server. It may be possible to use them instead - the feasibility of this is yet to be determined.